blocks/rich-text/editable.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Internal dependencies
+ */
+import RichText from './';
+
+class Editable extends RichText {
+	constructor() {
+		super( ...arguments );
+		// eslint-disable-next-line no-console
+		console.warn( 'Editable is deprecated use wp.blocks.RichText instead.' );
```

Comment:
Minor: this sentence could use a period. :)

Reply:
It already has a period?

Reply:
Yes, I did. :) Sorry, should have been more specific :)